### ISSUE TITLE
Add a way to list all packages managed by rua

### ISSUE THREAD

**Issue by gnzlbg** [vn971/rua#149]
It would be very helpful to be able to list all packages managed by rua, and list what's the currently installed version, as well as whether there is a newer verson available.

**Comment by teunvanson** [comment]
I second this. It is of course possible to see all foreign packages through Pamac, and perhaps also through Pacman (although I don't know how, since the packages installed through rua don't seem to have a repository name attached to them), but there currently seems to be no way of knowing whether rua is actually managing and updating them.

**Comment by virgilwashere** [comment]
> as well as whether there is a newer verson available.

That's covered by

```console
$ rua upgrade
Package            Current     Latest
git-sizer-bin      1.3.0-1     1.4.0-1
gitlab-glab-bin    1.16.0-1    1.17.0-1
libcgroup          0.41-2      2.0-1
rocketchat-desktop 3.2.0-3     3.2.2-1
rua                0.17.19-1   0.17.21-1

Do you wish to upgrade them? [O]=ok, [X]=exit.
```

Or even just `rua upgrade --printonly`

**Comment by vn971** [comment]
@virgilwashere not exactly, because `rua upgrade` only lists upgradable packages. This feature request is to list all non-pacman-managed packages I think.

WRT to this feature request, if anybody makes a clean PR for this feature, I'm open to have it in rua
